src/Compilers/Core/Portable/Generated/Operations.Generated.cs

```diff
@@ -3729,20 +3729,22 @@ public override IEnumerable<IOperation> Children
     internal sealed partial class UsingOperation : Operation, IUsingOperation
     {
         private IEnumerable<IOperation>? _lazyChildren;
-        internal UsingOperation(IOperation resources, IOperation body, ImmutableArray<ILocalSymbol> locals, bool isAsynchronous, IMethodSymbol? disposeMethod, SemanticModel? semanticModel, SyntaxNode syntax, bool isImplicit)
+        internal UsingOperation(IOperation resources, IOperation body, ImmutableArray<ILocalSymbol> locals, bool isAsynchronous, IMethodSymbol? disposeMethod, ImmutableArray<IArgumentOperation> disposeArguments, SemanticModel? semanticModel, SyntaxNode syntax, bool isImplicit)
             : base(semanticModel, syntax, isImplicit)
         {
             Resources = SetParentOperation(resources, this);
             Body = SetParentOperation(body, this);
             Locals = locals;
             IsAsynchronous = isAsynchronous;
             DisposeMethod = disposeMethod;
+            DisposeArguments = SetParentOperation(disposeArguments, this);
```

Comment:
>DisposeArguments = SetParentOperation(disposeArguments, this); [](start = 12, length = 62)

It looks like this sub-tree is getting different treatment by comparison to similar sub-trees for foreach. I think we should align the treatment and follow what we do for foreach.